README.md

```diff
@@ -44,10 +44,13 @@ class MyDjangoFSMModel(FreezableFSMModelMixin):
     NON_FROZEN_FIELDS = FreezableFSMModelMixin.NON_FROZEN_FIELDS + (
         'a_mutable_field',
     )
+    FSM_STATE_FIELD_NAME = 'state'  # defaults to 'state'
```

Comment:
I am thinking about the comment to mention the link between this value and the `FSMField`. Maybe something like:
```
# Assign this with the name of the`FSMField` if it is not 'state'. 
If your are using 'state' as the `FSMField` in your model, you can leave this one out.
See example in `mytest/models.py:FakeModel2`
```